modules/special-rules/pages/air-transport-mission.adoc

```diff
@@ -27,20 +34,27 @@ NOTE: If a {transport-x} Flyer unit is destroyed while airborne then any embarke
 
 . Make ranged attacks with the Flyer units, just like you would with ground units in the Shooting phase -- use the Firepower values of the Flyer units and any super-heavy weapons that they have.
 But note:
-* You can draw a line of fire only where an enemy unit lies within range *and* inside the forward 90° arc of your Flyer unit.
-Ignore any intervening terrain though, as it is far below the altitude of the flyers.
-* Otherwise, the attack works just like a normal shooting attack -- so for example,as long as you meet the other requirements then you may split fire between multiple target detachments.
+* You can attack only where an enemy unit lies within range *and* inside the forward 90° arc of your Flyer unit.
+Flyer units do not need line of fire, as they can see over all intervening units and terrain.
+* Otherwise, the attack works just like a normal shooting attack -- so for example, as long as you meet the other requirements then you may split fire between multiple target detachments.
 * *Halve the Firepower* of each {transport-x} Flyer unit that will disembark cargo units (and roll to lock on with any super-heavy weapons).
 . After you resolve the shooting effects, disembark the cargo units just like you would with a ground transportation unit.
-But note:
-* In each case, measure from the *centre* of the transport flyer's flight stand.
+//But note:
+//* In each case, measure from the *centre* of the transport flyer's flight stand.
 +
 {blank}[TODO: Are there any awkward edge cases where this 'measure from the centre' restriction poses a problem? If yes, what is the best alternative given the variety of base and model sizes that is even more variable than with ground units?]
-* Give the disembarked detachment whatever orders you prefer, except that you may not choose Overwatch or March orders.
-To be clear, you *may* put the detachment on Assault orders and make assault moves in the usual way this turn.
++
+{blank}[IJW reply: Yes, there are. A 60 mm round base (fairly common for stability purposes) makes it impossible to disembark units on 20 mm square or 25 mm round bases, and even strip bases barely squeeze in by a few mm. There are also 115 x 90 mm oval bases that are sometimes used for SM Landing Craft, or bigger Ork Landa conversions. I suggest using the standard disembarkation rules, and add a NOTE saying that landing Flyers can't be placed within 10 cm of enemy units and are Vehicles for terrain purposes.
++
+Plus there are players that use scenic bases rather than traditional flight stands, so there is no central column to measure from anyway.]
```

Comment:
I can't quite remember, but I might've had a half-baked idea that at this point you could just remove the model and measure from where the centre *was* a moment ago — given that you'd remove the model very soon anyway. But... other complications. So yeah, you're right, standard disembarkation rules. 👍 

Do we need to introduce a rule to prevent placement of transport flyers within 10 cm of their enemies though? I suggest it might be better simply to clarify how Snap Fire applies in this case? Perhaps: Yes, you can place transport flyers (and/or disembark units) within 10 cm of the enemy — but then Snap Fire will be in effect at that point. This would be more consistent with other cases wouldn't it, such as Drop Pods that happen to land within 10 cm of enemy units?

About treating landing flyers as Vehicle units for purposes of Dangerous Terrain tests: I suppose we should extend this to cover the possibility of future War Engine flyer units too? Hopefully it is common sense that a failure on the first roll of the test does nothing at all except lead to the second roll that may cause damage... though we might have to make it crystal clear that if the flyer is destroyed at this point, then normal rules for destroyed transports apply (as opposed to the 'everything dies' rule that applies while the transport flyer is actually in flight)?

Reply:
I don't think it's safe to use Drop Pods as a precedent—they land randomly, rather then being able to deliberately place them in a specific position.

In particular, if you're moving second I don't think it's a good idea to be able to place your disembarking units directly into base contact with the enemy, so that they're automatically in base contact in the Assault phase and therefore can't fail their assault move.

Reply:
Hmmm, I hadn't thought about of immediate base contact and thus avoidance of potential Leadership test to charge into contact. Good point.

Strictly speaking, units normally **can** come within 10 cm of the enemy during the movement phase. It's just that:

1. The opponent **may** invoke the Snap Fire rule. (In practice they always will, but the rules don't actually make it mandatory.)
2. If the opponent invokes the Snap Fire rule, then the player must 'bring the unit back' to the 10 cm threshold. This is the bit that's actually problematic for the air transport case.
3. Regardless, the player may not place units in base contact — as stated elsewhere in the rules, they may only do this as a result of a charge move in the Assault phase. So they can't use Air Transport to circumvent the Leadership test later, because at this point they can't place the units in base contact anyway.

So, strictly speaking again, it seems to me that we could easily allow players to place units within 10cm as a result of Air Transport — we'd just void point (2) in this case, re-affirm point (3), and apply the rest of the Snap Fire rule (rolling for hits) as usual (with a note that the usual destroyed transport rules apply when **landed** transport flyers are destroyed, as opposed to 'everyone dies' rule that applies when they are in flight). I'm not saying that we **should** necessarily do it like this — I'm just explaining my rationale for why I believe that we **could** if we think it best overall.

However, if you think 'may not place within 10 cm is better', then I have no objection. You only mention "landing Flyers" in your note — am I right to presume that the 10 cm exclusion zone should also apply to the cargo units that we disembark, not just the transport flyer units?

Reply:
> 3\. Regardless, the player may not place units in base contact — as stated elsewhere in the rules, they may only do this as a result of a charge move in the Assault phase. So they can't use Air Transport to circumvent the Leadership test later, because at this point they can't place the units in base contact anyway.

For what it's worth, the only restriction I can find is buried in _Assault moves and charges > Charge moves_ so is easily missed as a general rule. The current phrasing is also tied specifically to when you 'move' a unit:

> This is the only time that you may move your unit into base contact with an enemy unit.

So I can easily see players arguing that it doesn't apply to any text that says 'place' instead of 'move'. :-(


> (with a note that the usual destroyed transport rules apply when **landed** transport flyers are destroyed, as opposed to 'everyone dies' rule that applies when they are in flight)

I think I'm missing something. Is this still for transport flyers landing in dangerous terrain?

As far as I'm aware, Snap Fire shots happen before the transport flyer has landed, as the sequence is:

1. Move flyer on.
2. Resolve Snap Fire.
3. Flyer shoots (including being high up and seeing over intervening obstacles etc.).


5. Cargo units disembark.

> However, if you think 'may not place within 10 cm is better', then I have no objection. You only mention "landing Flyers" in your note — am I right to presume that the 10 cm exclusion zone should also apply to the cargo units that we disembark, not just the transport flyer units?

Doh, yes I meant the disembarking units as well!

Overall, I think a blanket ban on 10 cm is going to be safer, as it bypasses a lot of murky areas for rules interactions.